site/ja/tutorials/keras/basic_classification.ipynb

```diff
@@ -0,0 +1,1002 @@
+{
```

Comment:
<p>The library is already imported.</p><p><br></p><p>```suggestion</p><p>## ファッションMNISTデータセットのロード</p><p>```</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Yes. Your suggestion is right.